apps/safe-claiming-app/src/Widget.tsx

```diff
@@ -0,0 +1,191 @@
+
+const SpaceContent = styled("div")`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const Title = (props: TypographyProps) => (
+  <Typography
+    color="primary.main"
+    style={{ fontWeight: "bold", textAlign: "center" }}
+  >
+    {props.children}
+  </Typography>
+)
+
+const Subtitle = (props: TypographyProps) => (
+  <Typography
+    variant="subtitle2"
+    color="primary.light"
+    style={{ marginBottom: "16px", textAlign: "center" }}
+  >
+    {props.children}
+  </Typography>
+)
+
+const StyledButton = (props: ButtonProps) => (
+  <Button
+    size="large"
+    onClick={props.onClick}
+    variant="contained"
+    disableElevation
+  >
+    {props.children}
+  </Button>
+)
+
+const Widget = () => {
+  const [delegates] = useDelegatesFile()
+  const delegateAddressFromContract = useDelegate()
+  const balance = useTokenBalance()
+  const { safe } = useSafeAppsSDK()
+
+  const [vestings] = useAirdropFile()
+  const [userVesting, ecosystemVesting, investorVesting] = vestings
+
+  const totalAllocation = BigNumber.from(userVesting?.amount || 0)
+    .add(ecosystemVesting?.amount || 0)
+    .add(investorVesting?.amount || 0)
+
+  const currentDelegate = useMemo(() => {
+    if (delegateAddressFromContract) {
+      const registeredDelegateFromData = delegates.find((entry) =>
+        sameAddress(entry.address, delegateAddressFromContract)
+      )
+      return (
+        registeredDelegateFromData || { address: delegateAddressFromContract }
+      )
+    }
+  }, [delegateAddressFromContract, delegates])
+
+  const [userVestingStatus] = useFetchVestingStatus(
+    userVesting?.vestingId,
+    userVesting?.contract
+  )
+
+  const [ecosystemVestingStatus] = useFetchVestingStatus(
+    ecosystemVesting?.vestingId,
+    ecosystemVesting?.contract
+  )
+
+  const [investorVestingStatus] = useFetchVestingStatus(
+    investorVesting?.vestingId,
+    investorVesting?.contract
+  )
+
+  const totalClaimed = BigNumber.from(userVestingStatus?.amountClaimed || 0)
+    .add(ecosystemVestingStatus?.amountClaimed || 0)
+    .add(investorVestingStatus?.amountClaimed || 0)
+
+  const votingPower = useMemo(() => {
+    if (totalAllocation.gt(0) && totalClaimed.gt(0) && balance.gt(0)) {
+      return totalAllocation.sub(totalClaimed).add(balance)
+    }
+    return BigNumber.from(0)
+  }, [balance, totalAllocation, totalClaimed])
+
+  const handleClickClaim = () => {
+    const url = `${window.location.ancestorOrigins[0]}/apps?safe=${
+      safe.chainId === 1 ? "eth" : "gor"
+    }:${safe.safeAddress}&appUrl=${CLAIMING_DATA_URL}`
+
+    // @ts-ignore
+    window.top.location.href = url
+  }
+
+  const ctaWidget = (
+    <SpaceContent>
+      <div>
+        <Title>Become the part of Safe future!</Title>
+        <Subtitle>
+          Holding $SAFE is required to participate in governance.
+        </Subtitle>
+      </div>
+      <StyledButton>Buy Tokens</StyledButton>
```

Comment:
I would remove this button for now. I don't know if we want to advertise buying the token. 
Plus it is currently not transferable.

Reply:
I disabled the button and change the secondary text (see updated PR description)